modules/claims_api/app/controllers/claims_api/docs/metadata_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module Docs
+      class MetadataController < ::ApplicationController
+        skip_before_action(:authenticate)
+
+        def index
+          render json: {
+              meta: [
+                  {
+                    name: "Claims, Intent to File & Disability",
+                    version: "V1",
+                    internal_only: false,
+                    status: VERSION_STATUS[:rel],
+                    source: "https://api.va.gov/services/claims/docs/v1/api",
```

Comment:
source, meaning source of documentation? feels very similar to json api self link ..

Reply:
That's what I modeled it after, feel free to request a change, it's why I created this PR in the first place.